MapboxNavigation/RouteVoiceController.swift

```diff
@@ -272,7 +191,7 @@ public protocol VoiceControllerDelegate: class, UnimplementedLogging {
      - parameter synthesizer: the Speech engine that was used as the fallback.
      - parameter error: An error explaining the failure and its cause.
      */
-    func voiceController(_ voiceController: RouteVoiceController, didFallBackTo synthesizer: AVSpeechSynthesizer, error: SpeechError)
```

Comment:
This method feels like a dog-nail that we had to support `MapboxVoiceController`'s feature of a 'backup' synthesizer.

Reply:
That’s true to some extent, though I think the other consideration was that we wanted to send a telemetry event whenever speech got interrupted. There’s other ways of doing that, though. Do you think it would still be useful for the application to know when all the available speech synthesizers fail?

Reply:
For sure, knowing that speech synths had fail is important, but I thought that `SpeechSynthesizerController` describes a single speech Synth implementation, and thus it "doesn't know" about other implementations (so there is no such term as "fallback" in it's context). At this point, responsibility to manage multiple synths and notify when they all failed should be somewhere else. In current situation that is done by `MapboxSpeechSynthesizerController`, which will try to call all synths and if all of them fail - will report the latest error.